src/themes/bootstrap-table/bootstrap-table.scss

```diff
@@ -2,9 +2,9 @@
  * @author Dustin Utecht
  * https://github.com/wenzhixin/bootstrap-table/
  */
-@import './variables.scss';
+@import '_variables';
 @import '../theme.scss';
-@import './font.css';
+@import './_font.css';
```

Comment:
It doesn't need to be modified here.

Reply:
Didnt new that, reverted the import part.